### DISCUSSION TITLE
How do I embed required dll's and other files into one EXE?

### DISCUSSION THREAD

**Discussion by symonxdd** [Kinnara/ModernWpf#328] [Q&A]
When I drag the main program LauncherV to my desktop, it doesn't load up. I assume it's because it needs the dll's that my program uses.
The Release directory also has lots of localization folders which I really don't know what I should do with.
How can I bind these dll's, and any other needed files into one EXE file?
Also, what are the pdb files, are they required for proper functioning?
![image](https://user-images.githubusercontent.com/45405238/114251178-6435da00-99a0-11eb-8408-e6527a007c87.png)

**Comment by AZProductions** [comment]
@symonxdd You need to publish it in Visual Studio by right clicking on the project and pressing publish. Then create a new folder profile and edit it. Set it to be a stand alone **framework independent** program. Then publish it. Done!
